src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/payments/PaymentsPublisher.java

```diff
@@ -25,12 +26,16 @@
     private final QueueClient queueClient;
     private final ObjectMapper objectMapper;
 
+    private final int retryCount;
+
     public PaymentsPublisher(
         @Qualifier("payments") QueueClient queueClient,
-        ObjectMapper objectMapper
+        ObjectMapper objectMapper,
+        @Value("${azure.servicebus.payments.manual-retry-count}") int retryCount
```

Comment:
Maybe we should remove `manual`, as those retries aren't manual

Reply:
how about `max-retries` or `max-retry-count`?

Reply:
I want to distinguish automatic queue retry and our retry